### ISSUE TITLE
.NET Standard Support

### ISSUE THREAD

**Issue by nathanielcook** [cyotek/Cyotek.Collections.Generic.CircularBuffer#9]
Whenever you make your next build, please consider migrating to .NET Standard. There is an example [here](https://github.com/nieltg/Collections.Generic.CircularBuffer). Thanks!

**Comment by cyotek** [comment]
Hello,

That is firmly in my sights, especially as VS2017 makes it so incredibly easy to support multiple targets.

My only issue and the reason I haven't gone wholesale upgrading all the projects to do it is because I've been having a lot of difficulties with NuGet packaging. I can't use the built in "package on compile" stuff as I need to digitally sign the assemblies (and dual sign anything <=4 .0). However, I just cannot get nuget.exe to pack the csproj file - it always claims the binaries can't be found. No wonder as it will print out something silly like can't find file `directory\bin\debug` when the files are in `directory\bin\release\net47` as an example. Doesn't matter what kind of arguments I give it, helpful instructions on what paths to use. It's been driving me nuts actually. It will happily pack the nuspec file, but only if I remove all the tokens, which I don't want to do as then I invariably forget to update something. Oh yes, and also only if I manually tell it all the different files to include. Something else I don't want to have to manage when the csproj pack used to work perfectly fine. I dare say I'm missing something really obvious but nuget4 is giving me multiple issues (this isn't the only problem I have with it!) whilst nuget2 "just worked". Still, I'll figure it out eventually I'm sure.

Anyway long story short, yes, I fully intend on multi-targeting all binaries to whatever platforms I can.

Thanks for the suggestion!

Regards;
Richard Moss

**Comment by nieltg** [comment]
Hi @cyotek,
I have ported your project to .NET Standard 1.0 platform [here](https://github.com/nieltg/Collections.Generic.CircularBuffer).
I hope you can use this as a reference.

Thank you very much.

**EDIT:**
I'm sorry.
I didn't notice that I had been posting the same link with @nathanielcook's post.

**Comment by cyotek** [comment]
There is a now pre-release package available which offers various targets, including assorted flavours of .NET Standard. Took a while, but better late than never!
